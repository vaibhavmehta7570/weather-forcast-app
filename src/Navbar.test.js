import React from 'react'
import Navbar from './navbar'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'

describe('Should test navbar.js', () => {
  test('Should render navbar component correctly', () => {
    const navbarRendererResponse = renderer
      .create(
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>
      )
      .toJSON()
    expect(navbarRendererResponse).toMatchSnapshot()
  })
})
