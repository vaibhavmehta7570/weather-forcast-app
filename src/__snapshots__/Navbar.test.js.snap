// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test navbar.js Should render navbar component correctly 1`] = `
<nav
  className="navbar"
>
  <a
    href="/home"
    onClick={[Function]}
  >
    Home
  </a>
  <a
    href="/counter"
    onClick={[Function]}
  >
    Counter Task
  </a>
  <a
    href="/weatherapp"
    onClick={[Function]}
  >
    Weather App
  </a>
</nav>
`;
